soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3051,9 +3052,20 @@ void DrawRandoEditor(bool& open) {
     UIWidgets::Spacer(0);
 
     ImGui::Text("Seed");
-    if (ImGui::InputText("##RandomizerSeed", seedInputBuffer, MAX_SEED_BUFFER_SIZE, ImGuiInputTextFlags_CharsDecimal | ImGuiInputTextFlags_CallbackCharFilter, UIWidgets::TextFilters::FilterNumbers)) {
+    if (ImGui::InputText("##RandomizerSeed", seedInputBuffer, MAX_SEED_BUFFER_SIZE, ImGuiInputTextFlags_CharsNoBlank)) {
+        bool passthru = true;
         uint32_t seedInput;
-        ImGui::DataTypeApplyFromText(seedInputBuffer, ImGuiDataType_U32, &seedInput, "%u");
+
+        //Check for number-only string to pass through without hashing
+        for (char a = *seedInputBuffer; seedInputBuffer; a = *++seedInputBuffer) {
+            if (!(isdigit(a))) passthru = false; break;
```

Comment:
Not a fan of the double statements, especially on a single line if statement.

Better to just use curly braces.